k8s/sandbox/common/idam/idam-api.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: idam
   annotations:
     flux.weave.works/automated: "true"
```

Comment:
you might want to turn this off unless you want to pick up other preview changes as they come alone (if flux worked)

Reply:
i've specified an image name, so update to preview wouldnt deploy to sandbox ?